content/internal/procedure-version-control.md

```diff
@@ -219,6 +246,6 @@ Once the code has been tested, create a merge request on [Gitlab](https://gitlab
 
 ## README
 
-A `README` serves to describe the package, setup instructions, files  and any requirements the package may have. 
+A `README` serves to describe the package, setup instructions, files and any requirements the package may have. 
```

Comment:
Would it not be better to require a Markdown file? It seems unlikely that we'd actually write a raw text `README`.

Reply:
Done